demos/src/hover.mustache

```diff
@@ -0,0 +1,17 @@
+<div class="o-colors-page-background">
+	<div class='demo-tooltip-container' id="box-tooltip">
+
+		<button class='o-buttons o-buttons--big demo-tooltip-target' id="demo-tooltip-target">
+			Share
+		</div>
+
+		<div data-o-component="o-tooltip"
+		data-o-tooltip-position="right"
+		data-o-tooltip-target="demo-tooltip-target"
+		data-o-tooltip-show-on-hover="true">
```

Comment:
Nice links, thanks :-)

I also checked with the dataset accessor approach and it also returns strings.

``` js
> document.body
> <body>​…​</body>​
> document.body.dataset.oToolTipShowOnHover = true
> true
> document.body
> <body data-o-tool-tip-show-on-hover=​"true">​…​</body>​
> document.body.dataset.oToolTipShowOnHover
> "true"
```